src/blocks/image.js

```diff
@@ -4,6 +4,18 @@ import {EDIE_BLOCK_TYPE, EDIE_PROPS} from './common/formatDefinition';
 const imageToMjml = (item, blockRenderer, isTopLevelNode) => {
     let mjImage = toMjml('mj-image', item.properties, EDIE_PROPS[EDIE_BLOCK_TYPE.IMAGE]);
 
+    if (item.properties.condition) {
+        const open = item.properties.condition.indexOf(' ') >= 0
+            ? item.properties.condition.split(' ')[0]
```

Comment:
Really just the first part of string before space? How would that work?

Reply:
Omg... thanks for catching this ;))) It was supposed to support:
{{#something with args}} .... {{/something}} 

Fixed ;)